packages/sdk/typescript/human-protocol-sdk/src/storage.ts

```diff
@@ -55,6 +58,30 @@ export default class StorageClient {
     );
   }
 
+  /**
+   * **Download files from cloud storage.*
+   *
+   * @param {string} url - URL to the file
+   * @returns {Promise<File>} - Downloaded file
+   */
+  public async downloadFileFromUrl(url: string): Promise<File> {
```

Comment:
If you add this method inside StorageClient class, you must send endPoint and useSSL parameters to constructor. I think this should be a static method outside of StorageClient